FAIL
run: wait(exec("multi-recurse 13")) returned -1: FAILED
